api/src/changeRequests/processChangeRequest.ts

```diff
@@ -80,37 +80,38 @@ export async function processChangeRequest(
     // Insert / update the document in the database
     const upsertResult = await db.upsertDoc(doc);
 
+    // ******************* tmp disable changeDoc, until it will be implimented on the CMS (ATM the changeDoc's only slow down the api) ********************
```

Comment:
As for this comment, can you link to a Github issue that will change this back? Then it's easy in the future to see if this comment is still relevant. "tmp" might not be so temporary 😅 Also I don't think it needs to be so extensive and attention drawing but that might just be personal preference 😄

```suggestion
    // TODO: Reactivate change diffs in change requests - https://github.com/bccsa/luminary/issues/...
```